cachi2/core/package_managers/bundler/main.py

```diff
@@ -84,7 +84,7 @@ def _get_name_and_version_from_lockfile(dependencies: ParseResult) -> Optional[t
     https://github.com/containerbuildsystem/cachi2/blob/main/docs/design/bundler.md
     """
     for dep in dependencies:
-        if isinstance(dep, PathDependency) and dep.path == ".":
+        if isinstance(dep, PathDependency) and dep.subpath == ".":
```

Comment:
This renaming change should be squashed into commit https://github.com/containerbuildsystem/cachi2/pull/659/commits/c4763e53121bbdd5db5ce69b62928738be679e2f and just listed as one of the renamings done by the patch.